app/models/exercise.rb

```diff
@@ -50,7 +50,7 @@ def self.for(track_slug, exercise_slug)
     joins(:track).find_by('tracks.slug': track_slug, slug: exercise_slug)
   end
 
-  delegate :files_for_editor, :exemplar_files, :introduction, :instructions, :source, :source_url, to: :git
+  delegate :files_for_editor, :exemplar_files, :introduction, :instructions, :source, :source_url, :dir, to: :git
```

Comment:
```suggestion
  delegate :files_for_editor, :exemplar_files, :introduction, :instructions, :source, :source_url, to: :git
  delegate :dir, to: :git, prefix: true
```

Reply:
I think I prefer the clarity of `exercise.git_dir` rather than `exercise.dir` here.